api/src/main/java/org/pmiops/workbench/db/dao/UserDao.java

```diff
@@ -102,8 +102,6 @@
 
     void setInstitutionName(String institutionName);
 
-    Timestamp getFirstRegistrationCompletionTime();
```

Comment:
We have not been populating this since Access Tiers were added.  We still have this info (and we are still populating the WRD with this) but it's not here.  It's in `user_access_tier.first_enabled`